### DISCUSSION TITLE
Reassess the definition of "2FA" to include things that are better than _just passwords_

### DISCUSSION THREAD

**Discussion by skyzyx** [2factorauth/twofactorauth#7085] [Ideas]
- [x] I have searched [open issues and pull requests](https://github.com/2factorauth/twofactorauth/issues?q=is%3Aopen). The issue I'm creating is not a duplicate of an existing open issue or pull request.

### Information about the feature to be added: ###

There are a lot of complex ideas to unpack here, but I'll try to be brief. These are, I believe, _fundamental truths_ about authentication.

* The human mind is the worst place for storing passwords.
* The human mind is the worst tool for generating passwords.
* Using single-factor auth which avoids passwords is _better_ than using passwords as that single factor.

Therefore, I would like to respectfully request that the definition of "2FA" used by this site _evolve_ from the literal definition to something that allows sites to do _better_ things, even if it's still technically single factor.

* One of these examples is Passkeys, which can be used _in place of_ username + password.

* Another of these is "passwordless" authentication. I've seen the Contribution Guide, and understand the use of the literal definition of _two-factor_, but this is something I'd request we redefine.

    * There is no perfect "silver bullet" here, and I'm not asking we try to synthesize one.

    * Maybe separate passwordless-via-email (where most email providers have a web-facing password login, and therefore are some definition of "less secure") from passwordless-via-app (using a phone/tablet is more personal, and does not have a web-facing password login, and therefore is some definition of "more secure").

_Are we trying to list sites with literal multi-factor authentication, or are we trying to provide information that can inform and enable users to make themselves more secure?_

If it's the former, then nevermind. But if it's the latter, I'd love to see a more comprehensive "better than the status quo" listing that includes secure _alternatives_ to literal 2FA (where 2FA is _one_ solution to the problem, but not treated as the _only_ solution to the problem).

Thanks for considering.

**Comment by Carlgo11** [comment]
@skyzyx there's lots of websites spreading the word on online security and authentication solutions. 2fa.directory spreads awareness about one of those solutions, multi factor authentication.
Whilst we don't want to negate the usefulness of non-password single authentications, it's not something that we're going to list here on 2fa.directory.
You could however fork our repository and make your own project if you're passionate about 1FA solutions. [Dongleauth.com](http://dongleauth.com/) did just that when they wanted more specialized hardware content.
